server/StudentList.ts

```diff
@@ -37,5 +37,9 @@ class StudentList extends EventEmitter{
         return new StudentList();
     }
 
+    compareTwoStudents(student1: Student, student2 : Student) : boolean {
+        return (student1.name == student2.name && student1.surname == student2.surname && student1.nick == student2.nick)
```

Comment:
chyba wystarczy sprawdzić samo nick, żeby się nie było dwóch użytkowników z tym samy nickiem

Reply:
zalecenie z góry

Reply:
We need to decide what to do with **surname** and **name**. For now, it doesn't harm if we check all object values.